cfme/tests/configure/test_multi_tenancy_manual.py

```diff
@@ -0,0 +1,438 @@
+# -*- coding: utf-8 -*-
+# pylint: skip-file
+"""Manual tests"""
+
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [pytest.mark.ignore_stream('5.9', '5.10', 'upstream')]
```

Comment:
Need to collect against at least one stream

Reply:
All test applicable to both 5.9 and 5.10, so removing both.